floater/generators.py

```diff
@@ -97,13 +97,34 @@ def parcel_area(self, latlon=True):
             return dx * dy
 
 
-    def to_mitgcm_format(self, filename, tstart=0, mesh='rect'):
-    	#xx, yy = np.meshgrid(x, y)
+    def to_mitgcm_format(self, filename, tstart=0, iup=0, mesh='rect'):
+	"""Output floatset in MITgcm format
+	PARAMETERS
+	----------
+	filename : The filename to save the floatset data in 
+		   (e.g.float.ini.pos.hex.bin)
+	tstart : time for float initialisation (default = 0)
+	iup : flag if the float
+         - should profile ( > 0 = return cycle (in s) to surface)
+         - remain at depth ( = 0 )
+         - is a 3D float ( = -1 )
+         - should be advected WITHOUT additional noise (= -2 ); 
+	this implies that the float is non-profiling
+         - is a mooring ( = -3 ); i.e. the float is not advected
+	mesh : choice of mesh
+	 - 'rect' : rectangular cartesian
+	 - 'hex' : hexagonal
+	"""	
```

Comment:
You can look at the existing tests:
https://github.com/rabernat/floater/blob/master/floater/test/test_generators.py

The simplest thing to do is just call the function and make sure it doesn't error.

A more thorough test would make sure the contents of the generated file are correct.